tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/txprune/HBaseTransactionPruningPlugin.java

```diff
@@ -335,6 +351,25 @@ private long computePruneUpperBound(TimeRegions timeRegions) throws IOException
     return -1;
   }
 
+  private SortedSet<byte[]> filterDeletedRegions(final Set<TableName> existingTables,
```

Comment:
Better name this filterDeletedTables(), or filterDeletedTableRegions()
- it does not filter out deleted regions
- it does filter out all regions of deleted tables